adapter/internal/api/apis_impl.go

```diff
@@ -231,8 +232,7 @@ func verifyMandatoryFields(apiJSON config.APIJsonData) error {
 
 // ApplyAPIProjectFromAPIM accepts an apictl project (as a byte array), list of vhosts with respective environments
 // and updates the xds servers based upon the content.
-func ApplyAPIProjectFromAPIM(payload []byte, vhostToEnvsMap map[string][]string) ([]*notifier.DeployedAPIRevision,error) {
-	var deployedRevisionList []*notifier.DeployedAPIRevision
+func ApplyAPIProjectFromAPIM(payload []byte, vhostToEnvsMap map[string][]string) (deployedRevisionList []*notifier.DeployedAPIRevision, err error) {
```

Comment:
shall we fix the formatting issue (line length)

Reply:
Fixed with https://github.com/wso2/product-microgateway/pull/2482